packages/front-end/components/Features/SDKConnections/SDKLanguageLogo.tsx

```diff
@@ -176,23 +124,50 @@ export default function SDKLanguageLogo({
   showLabel = false,
   size = 25,
   titlePrefix = "",
+  version,
 }: {
   language: SDKLanguage;
   showLabel?: boolean;
   size?: number;
   titlePrefix?: string;
+  version?: string;
```

Comment:
Is this latest version, users version, or something else?

Reply:
whatever you want it to be. The language logo is a reusable visible component that you can plug in a version number for and it will render it.